packages/components/src/modal/index.tsx

```diff
@@ -371,6 +361,7 @@ function UnforwardedModal(
  * ```jsx
  * import { Button, Modal } from '@wordpress/components';
  * import { useState } from '@wordpress/element';
+  import { withIgnoreIMEEvents } from '../utils/with-ignore-ime-events';
```

Comment:
This is unnecessary, right?

Reply:
Oops, VS Code sometimes gets confused and auto-imports stuff in the JSDoc code snippet 😅 Fixed!